_includes/head.html

```diff
@@ -63,7 +63,7 @@
   }
 </script>
 
-<script src="{{ 'js/jquery.min.js' | relative_url }}"></script>
```

Comment:
Can we delete the `js/jquery.min.js` file?

Reply:
yup, let me delete it